app/controllers/concerns/sufia/users_controller_behavior.rb

```diff
@@ -142,4 +138,11 @@ def get_sort
            end
     return sort_val
   end
+
+  def deny_access(exception)
+    unless @user == current_user
```

Comment:
What is this `unless` adding?  If we've already determined that the current user lacks the ability to edit/update `@user` on line 10, why the `unless`?